recipe/meta.yaml

```diff
@@ -28,25 +35,21 @@ requirements:
     - perl
     - nasm
     - make
-    # m2-git is required, because "git clone" won't work when combining the "git" package with "m2-bash"
-    - {{ posix }}git  # [win]
     - {{ posix }}bash  # [win]
     - {{ posix }}coreutils  # [win]
     - {{ posix }}grep  # [win]
     - {{ posix }}sed  # [win]
-  host:
```

Comment:
An empty host section was often used to enforce that recipes are detected as cb3, not sure if this is relevant here.